test/ctia/task/migration/store_test.clj

```diff
@@ -875,3 +889,85 @@
           (doseq [store stores]
             (is (nil? (get-in store [:source :store])))
             (is (nil? (get-in store [:target :store])))))))))
+
+(deftest target-store-properties-test
+  (let [default-es-props {:host "localhost"
+                          :port 9200
+                          :transport "http"
+                          :indexname "ctia_default"
+                          :replicas 1
+                          :refresh_interval "1s"
+                          :default_operator "AND"
+                          :shards 5
+                          :refresh false
+                          :rollover {:max_docs 10000000}
+                          :aliased true}
+
+
+        source-custom-props {:indexname "source-indexname"
+                             :shards 4}
+
+        migration-cluster-props {:host "es7.iroh.site"
+                                 :port 443
+                                 :transport "https"}
+
+        target-store-props {:indexname "custom-target-indexname"
+                            :shards 4
+                            :refresh_interval "10s"}
+        base-es-props {:ctia {:store {:es {:default default-es-props}}}}
+        with-source-props (assoc-in base-es-props
+                                    [:ctia :store :es :sighting]
+                                    source-custom-props)
+        with-migration-cluster-props (assoc-in with-source-props
+                                               [:ctia :migration :store :es :default]
+                                               migration-cluster-props)]
+
+    (testing "simple migration properties that preserves ES source properties and simply adds a prefix to newly created index"
```

Comment:
Ideally `testing`s are complete sentences, e.g. subject + predicate. Without a predicate they feel incomplete, as the behavior being tested is not described.